modules/ROOT/pages/setup-payload-storage-API.adoc

```diff
@@ -1,29 +1,33 @@
 = Payload Storage API for Partner Manager
 
-Anypoint Partner Manager enables you to configure your own storage solution, such as database, Amazon S3, Azure Blob Storage, and so on to store transmission content, such as incoming and outgoing B2B message payloads.  
+Anypoint Partner Manager enables you to configure your own storage solution (for example, a database, Amazon S3, or Azure Blob Storage) to store transmission content, such as incoming and outgoing B2B message payloads.
 
-Before you create your first trading partner and message flow, you must configure your payload storage API within the host organization in Partner Manager.
+Before you create your first xref:configure-partner#create-partner[trading partner] and xref:configure-message-flow[message flow], you must configure your payload storage API within the host organization in Partner Manager.
```

Comment:
This sentence alone could be the transition to the steps....

Reply:
Done